client/src/components/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import { Outlet } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import './Header.css';
+export function Header() {
+  return (
+    <>
+      <div className="row colors">
+        <Link to="/">
+          <img src="/TherapyLogo.png" className="logo" alt="Therapy Logo" />
+        </Link>
+        <div className="nav-links">
+          <Link to="/sign-in">
+            <button className="my-butt">Sign In</button>
```

Comment:
You wouldn't usually want a button inside a Link. It is technically illegal in HTML. If you want the link to _look_ like a button, you can use CSS to get that. Or you could just remove `Link` and use `navigate()` in the button's `onClick`.